.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,27 @@
+name: NPM Publish
+
+on:
+  [workflow_call, workflow_dispatch]:
```

Comment:
As you can see from the failed check, this can't be an array. The two events are their own keys in the "on" object.

Reply:
I swear I referenced documentation for this! https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#using-multiple-events

I will change it to events on newline.

Reply:
Ah I see, you can only use array syntax if there is no yaml data nested underneath. Ok, updated.

Reply:
Ah, that makes sense! Yeah, I didn't realize you could even DO the array method there. Docs are obviously not clear on that, so not your fault for thinking it'd be fine.